src/events/crawler/lib/isoDate.js

```diff
@@ -0,0 +1,54 @@
+﻿/* eslint-disable camelcase */
+
+import { ZonedDateTime, LocalDateTime, LocalDate, ZoneId } from '@js-joda/core';
+import '@js-joda/timezone/dist/js-joda-timezone-10-year-range'; // minimize package size by only importing tz data for current year ±5 yrs
+
+// util functions
+const currentJsDate = () => new Date(Date.now()); // allows us to mock current date
```

Comment:
I'd use UTC for now and today. What is the point in having different times based on a server's or client computer's locale settings.

I recommend making the codebase 100% UTC, with conversions to and from the location's timezone where needed.

Reply:
I mean I agree with the joda object not having a timezone. It's just that the "local" time we are working with should always be UTC local.

Reply:
yeah, that's probably the right call.